### PR TITLE
Added id

### PR DESCRIPTION
**Pull request by henryk86**

### CODE REVIEW

File: yaml/10001488.yaml [hunk 1]
```diff
@@ -27,4 +27,9 @@ data:
   - Der Tag der Prüfung
   - Nasri gibt nicht auf!
   - Auf dem Rücken des Elefanten
-  ids: []
+  ids:
+  - audio-id: 1679659474
+    hash: 589662E9A0708ED98893278646622DC6D7CF5EA8
```
Comment (SciLor) [inline yaml/10001488.yaml:32]: please change it to lowercase

File: yaml/11000604.yaml [hunk 2]
```diff
@@ -36,4 +36,9 @@ data:
   - Tilly, die Taube
   - In der Stadt (Das Geräuschelied)
   - Wenn die Stadt schlafen geht
-  ids: []
+  ids:
+  - audio-id: 1718011946
+    hash: 4F593350306E3DAAA9166EC2726E5DCEA936327F
```
Comment (SciLor) [inline yaml/11000604.yaml:41]: please change it to lowercase

Comment (henryk86) [reply yaml/11000604.yaml]: done

File: yaml/10001488.yaml (hunk 1 above)
Comment (henryk86) [reply yaml/10001488.yaml]: done

**Pull request merged**